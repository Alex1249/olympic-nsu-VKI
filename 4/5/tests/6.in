70 58
-426 669 329 243 768 142 -79 640 776 685 790 -511 47 596 724 5 758 -624 531 632 417 -499 979 -69 498 412 701 923 -843 586 -48 -422 -727 -869 336 213 -218 189 882 636 -661 -33 146 -786 3 -97 725 -444 470 -492 687 96 -547 -891 -991 291 -609 -244 542 -780 -262 264 43 439 -611 797 -169 -531 107 197
1 703
0 1372
0 1279
1 717
1 1064
1 2225
0 1021
0 2006
0 529
0 1399
1 1877
1 944
0 2094
0 1588
1 395
1 834
0 1717
0 1621
1 807
0 1948
0 353
1 1193
0 2232
1 301
0 1072
1 804
0 1740
1 1293
0 421
1 413
0 954
0 628
0 1031
1 848
0 1049
1 257
1 1632
1 541
1 1606
1 1972
0 1599
0 390
0 1825
0 756
1 1719
1 309
1 1424
0 842
1 1757
0 1012
1 1340
1 676
0 206
0 691
0 2191
1 467
1 2023
1 1350
