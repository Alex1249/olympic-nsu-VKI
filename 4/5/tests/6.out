-426 669 329 243 768 142 -131151 640 1073742600 263853 790 -1073742335 1048623 596 4820 5 758 -624 531 632 417 -4595 263123 -2117 498 150995356 536904381 923 -843 33354 -48 -2470 -16777943 -33637 336 213 -218 4194493 882 636 -661 -33 33554578 -786 32771 -97 725 -444 470 -2540 33555119 -2147483552 -547 -1659 -525275 291 -609 -1073742068 1566 -780 -270 2312 43 16777655 -611 797 -169 -531 107 16453 
